protocol-units/bridge/integration-tests/tests/eth_movement.rs

```diff
@@ -3,12 +3,163 @@ use alloy::{
+			.await
+			.context("Failed to get Alice's account balance")?
+	);
+	println!(
+		"Bob: {:?}",
+		coin_client
+			.get_account_balance(&bob.address())
+			.await
+			.context("Failed to get Bob's account balance")?
+	);
+	child.kill().await.context("Failed to kill the child process")?;
+	//let _ = child.wait().await.expect("Failed to wait on process termination");
+
+	Ok(())
+}
+
+#[tokio::test]
+//#[ignore]
+async fn test_movement_client_should_publish_package() -> Result<(), anyhow::Error> {
+
+	// at beginning of test, check whether .movement is present with a conditional statement so it works either way
+	// at end of test, delete .movement
+
+	let _ = tracing_subscriber::fmt()
+        .with_max_level(tracing::Level::DEBUG)
+        .try_init();
+	
+	let (mut harness, mut child) = TestHarness::new_with_movement().await;
+	{ let movement_client = harness.movement_client_mut().expect("Failed to get MovementClient");
+
+	let _ = movement_client.publish_for_test();
+	}
+
+	child.kill().await.context("Failed to kill the child process")?;
+	
+	Ok(())
+}
+
+#[tokio::test]
+#[ignore]
+async fn test_movement_client_should_successfully_call_lock() -> Result<(), anyhow::Error> {
+
+	let _ = tracing_subscriber::fmt()
+        .with_max_level(tracing::Level::DEBUG)
+        .try_init();
+	
+	let (mut harness, mut child) = TestHarness::new_with_movement().await;
+	{ let movement_client = harness.movement_client_mut().expect("Failed to get MovementClient");
+
+	let _ = movement_client.publish_for_test();
+	}
+	
+        let initiator = b"0x123".to_vec(); //In real world this would be an ethereum address
+        let recipient:MovementAddress = MovementAddress(AccountAddress::new(*b"0x00000000000000000000000000face"));    
+        let bridge_transfer_id = *b"00000000000000000000000transfer1";
+        //let pre_image = b"secret".to_vec();
+        let hash_lock = *keccak256(b"secret".to_vec()); 
+        let time_lock = 3600;
+        let amount = 100;
+	
+	harness
+	.movement_client_mut()
+	.expect("Failed to get MovmentClient")
+	.lock_bridge_transfer_assets(
+		BridgeTransferId(bridge_transfer_id),		
+		HashLock(hash_lock),
+		TimeLock(time_lock),
+		InitiatorAddress(initiator),
+		RecipientAddress(recipient),
+		Amount(amount), // Eth
+	)
+	.await
+	.expect("Failed to complete bridge transfer");
+
+	harness
+	.movement_client_mut()
+	.expect("Failed to get MovmentClient")
+	.complete_bridge_transfer(
+		BridgeTransferId(bridge_transfer_id),
+		HashLockPreImage(b"secret".to_vec()),
+	)
+	.await
+	.expect("Failed to complete bridge transfer");
+
+	child.kill().await.context("Failed to kill the child process")?;
+	//let _ = child.wait().await.expect("Failed to wait on process termination");
+	Ok(())
+}
+
+#[tokio::test]
+async fn test_movement_client_complete_bridge_transfer() -> Result<(), anyhow::Error> {
+	
+	Ok(())
+}
+
+#[tokio::test]
+async fn test_movement_client_abort_bridge_transfer() -> Result<(), anyhow::Error> {
+	
+	Ok(())
+}
 
```

Comment:
Might want to add a test complete and text cancel as well

Reply:
This PR scope has been reduced to `test_movement_client_should_publish_package` due to complexities that emerged with `test_movement_client_should_successfully_call_lock`.

I'll do `test_movement_client_should_successfully_call_lock` so it's up and running ideally today in a separate PR. (And then the test complete and test cancel, maybe those could go in the next PR as well).